src/measure_theory/measure_space.lean

```diff
@@ -974,6 +974,10 @@ by { rw [restrict, restrictₗ], convert le_lift_linear_apply _ t, simp }
   (μ.restrict t).restrict s = μ.restrict (s ∩ t) :=
 ext $ λ u hu, by simp [*, set.inter_assoc]
 
+lemma rectrict_comm (hs : measurable_set s) (ht : measurable_set t) :
+  (μ.restrict t).restrict s = (μ.restrict s).restrict t :=
```

Comment:
It might be worth stating a version of this in terms of https://leanprover-community.github.io/mathlib_docs/init/logic.html#right_commutative

Reply:
Something like `left_commutative (restrict : measure α → set α → measure α)`

Reply:
also worth fixing the name typo :b

Reply:
I think my comment is nonsense as it ignores `hs` and `ht`.